### ISSUE TITLE
Status of this project

### ISSUE THREAD

**Issue by azerothl** [linkedin/dustjs#779]
Hi, I'm using dustjs in a lot of my projects and would apreciate if someone could make a point on the status of this project, is it still maintained ? Is there a clear roadmap ?

Thanks

**Comment by Keyes** [comment]
An official statement on the status of dustjs would be highly apreciated @jimmyhchan @sethkinast @vybs .
We're using dustjs in our company as out main templating system, but due to the inactivity of this project we unfortunately might switch to something else soon :/

**Comment by ceastman-ibm** [comment]
Yes status update would be good on this project, we use it also: @jimmyhchan @sethkinast @vybs

**Comment by zenoven** [comment]
any update ?

The stream support is awesome and that is the reason I want to try this project, but if it's not maintained any more I'll have to try another one

**Comment by Nanonid** [comment]
From another comment, deprecated by linkedin.  An "official" fork is probably in order.

**Comment by gilsondelrei** [comment]
I have used dustjs in my projects, but considering that Linkedin has
deprecated it, i'll try Markojs in place.of Dustjs.


Em qua, 31 de out de 2018 às 13:52, Nanonid <notifications@github.com>
escreveu:
